module-apps/apps-common/widgets/TimeFixedWidget.hpp

```diff
@@ -3,40 +3,29 @@
 
 #pragma once
 
+#include "DigitsContainer.hpp"
 #include <Text.hpp>
 #include <BoxLayout.hpp>
 #include "widgets/DateWidget.hpp"
 #include <time/FromTillDate.hpp>
 
 namespace gui
 {
-    struct DimensionsParams
-    {
-        std::uint32_t mainBoxHeight;
-        std::uint32_t mainBoxWidth;
-        std::uint32_t digitMaxWidth;
-        std::uint32_t colonWidth;
-        std::uint32_t minusWidth;
-        std::uint32_t leftBoxWidth;
-        std::uint32_t rightBoxWidth;
-    };
 
     class TimeFixedWidget : public Rect
     {
       public:
-        template <size_t N> using DigitsContainer = std::array<Label *, N>;
-
         struct LeftBox
         {
             HBox *box    = nullptr;
             Label *minus = nullptr;
-            DigitsContainer<3> digits{nullptr};
+            DigitsContainer<3> container{nullptr};
```

Comment:
Initialising it with `nullptr` is a bit misleading here for me. Imho, it'd be better to use default constructor if you want to do such things as this is definitely not a proper way to initialise `std::array<Label *, N>`